MyCryptoHQ/MyCrypto
# Production Release Changes

Closes #1542
Closes https://github.com/MyCryptoHQ/MyCrypto/issues/1743

### Description

Preps the site for being in the limelight on production. Should only be merged once TODOs are addressed, and we're ready to launch.

### Changes

* Add a welcome modal that shows when you
* Remove beta agreement blocker & announcement from the top of the site
* Add a link to "MyCrypto Classic" to footer
* Move onboard modal into Root.tsx with the rest of the global modals
* Give tooltips the ability to point in any direction, point online status tooltip to the left
* Fixes modal fade on modals with no footer
* Removes entirely unused `TitleBar` component from Root.tsx
    * This was a holdover from an earlier electron UI that I forgot to clean up
* Update README.md

### TODOs

* [ ] Link to published blog post about the launch
* [ ] Setup `https://legacy.mycrypto.com` (cc @skubakdj)
    * Also make sure `https://legacy.mycrypto.com/helpers` still works too
* [ ] Get copy approved by Jordan
* [ ] (Maybe) Get a better welcome modal image / banner

### Steps to Test

1. Open site and confirm you get no welcome modal
2. Add a `gasPrice` entry of anything to local storage via chrome devtools
3. Refresh the page and confirm you get a welcome modal
4. Continue through the modal and refresh the page, confirm you do not get a welcome modal

^ Note that this won't work on downloadable build / mycryptobuilds, since that's the downloadable HTML build (which has onboarding / welcome modals disabled)

## Screenshots

### Welcome Modal

<img width="582" alt="screen shot 2018-04-25 at 3 42 39 pm" src="https://user-images.githubusercontent.com/649992/39268969-528fde54-489f-11e8-8a27-7421fed8f801.png">



### Welcome Modal w/ Beta Local Storage

<img width="694" alt="screen shot 2018-04-25 at 3 42 28 pm" src="https://user-images.githubusercontent.com/649992/39268977-55ef90f8-489f-11e8-9e56-b160b4bcc0e9.png">
